hmt_escrow_js/manifest-schema-pydantic.js

```diff
@@ -0,0 +1,602 @@
+module.exports = {
```

Comment:
Nice! Can you quickly throw in a comment in the readme on how you generated this?

Reply:
done